### ISSUE TITLE
Blank options passed to sub jobs as literal strings, RD 2.2.3

### ISSUE THREAD

**Issue by richiereynolds** [rundeck/rundeck#992]
Hi, I think this might be a bug:
Job AddNode includes an optional multi select option "Attributes".
AddNode calls the AddAttributesToNode job which also has an optional multi select option "Attributes".
Attributes is passed to the AddAttributesToNode job in the normal manner as ${option.Attributes}
If the user didn't add any attributes to the AddNode job, then Attributes gets passed to the AddAttributesToNode job as the literal string " ${option.Attributes}" rather than a blank value.

e.g. in an inline script in the AddAttributesToNode job, @option.Attributes@, gets replaced by the literal string ${option.Attributes}

Thanks, Richard.

**Comment by ScottG489** [comment]
I can still reproduce this issue on version **2.6.0-1**

Steps to reproduce:
1. Create new job
2. Add a option, leaving all properties of it at their defaults (not required, no default value, etc)
3. Add a Local Command step which echo's the option. In my case I called my option '**field**':
`echo ${option.field}`
4. Finish creating the job and navigate to it to run it
5. Run the job without entering anything into the text box

The output will be `${option.field}` literally

Not sure if this is a regression in functionality between the version it was allegedly fixed in (**2.5.2**) and my current version (**2.6.0-1**) but it's definitely still broken. I can create a new issue if that would be more appropriate.

Although this is the issue that was referenced as being fixed by commit [4dd190e](https://github.com/rundeck/rundeck/commit/4dd190e17a590f5730e782f646c4b729d2f5ca96) I also posted this in response to #1272 since it is the issue that was referenced as being fixed in your [release notes](http://rundeck.org/news/2015/07/07/rundeck-2.5.2.html#issues)
